docs/introduction.mdx

```diff
@@ -20,7 +20,7 @@ The [emotion](https://www.npmjs.com/package/emotion) package is framework agnost
 
 - Requires no additional setup, babel plugin, or other config changes.
 
-- Works in situations where configuration is restricted or not possible such as with [Create React App](https://facebook.github.io/create-react-app)
+- Works in situations where configuration is restricted or not possible such as with [Create React App](https://facebook.github.io/create-react-app). However, it should be noted that with the current version of `Create React App` as of October 2019, the below methods also work out of the box for both the default and `--typescript` configurations.
```

Comment:
I'm not sure if this change is needed. Could you explain in what way it was confusing before the change?